packages/components/src/input-control/index.tsx

```diff
@@ -28,6 +30,18 @@ function useUniqueId( idProp?: string ) {
 	return idProp || id;
 }
 
+const WithTooltip = ( { showTooltip, text, children }: WithTooltipProps ) => {
+	if ( showTooltip && text ) {
+		return (
+			<Tooltip text={ text } position="top center">
+				<TooltipWrapper>{ children }</TooltipWrapper>
```

Comment:
I found directly wrapping the `InputField` in the `Tooltip` didn't render. Following the approach noted in https://github.com/WordPress/gutenberg/pull/24966#issuecomment-685875026 which adds a `div` around the `Tooltip` children works.

Reply:
Maybe we could add an inline comment about this?